src/backend/egl/display.rs

```diff
@@ -410,6 +410,7 @@ impl EGLDisplay {
     }
 
     /// Exports an [`EGLImage`] as a [`Dmabuf`]
+    #[allow(clippy::not_unsafe_ptr_arg_deref)]
```

Comment:
Why did that trigger?

Reply:
Rust 1.60, EGLImage is typedefed to a pointer.

Reply:
From Matrix:

> well our EGL code is not really managing unsafe properly tbh.
And EGLImage is just a typedef.
>
> I think it needs an overhaul anyway. So for now its fine, I guess.
>
> If you add a #allow-annotation just make sure to add a TODO somewhere as well.